gen/container_test.go

```diff
@@ -352,6 +427,13 @@ func TestCrazyTown(t *testing.T) {
 					{4, 5, 6},
 				},
 			},
+			tc.ContainersOfContainers{
+				ListOfLists: [][]int32{
+					{1, 2, 3},
+					{4, 5, 6},
+					{7, 8, 9},
+				},
```

Comment:
The tests for Container and Typedefs are collating the round trip and Equals
tests. Can we pull them into their own `Test*` functions? The tests are hard
to read as it is; adding a second struct that is just "not equal" to the
struct being parsed is weird. You can leave the "x is equal to x" assert there
as a sanity check, though.